cli/src/cli.rs

```diff
@@ -432,6 +441,13 @@ impl Command {
 
                 println!("{}", code);
             }
+
+            Command::Sign { args } => {
+                let signed_msg = args
+                    .private_key
+                    .sign_ecdsa(Message::from_digest(keccak(&args.msg).into()));
+                println!("{signed_msg}");
```

Comment:
This is not the expected signature. For Ethereum message signing, we have the [ERC-191](https://eips.ethereum.org/EIPS/eip-191) that specifies how messages should be signed.

Reply:
In addition to the above, we want the signed message to be printed with the prefix `0x`.